packages/admin-next/dashboard/src/routes/categories/common/components/category-tree/category-tree.tsx

```diff
@@ -0,0 +1,135 @@
+import { ProductCategory } from "@medusajs/medusa"
```

Comment:
A few things I noticed with the tree:

1. Sometimes when an item is moved around, and then you click on Edit in the Action menu, the tree just rerenders and the edit modal is not open

2. toggling one item sometimes toggle another item

https://github.com/medusajs/medusa/assets/16856471/d54c6419-b638-4c13-8d7d-74408e73a71f

Reply:
I can't reproduce this locally, but I can see it on the preview. I think this stems from the staging server being very slow, and you can end up performing actions before the ordering has propagated correctly. I'll add some logic to disable the nodes while the server is still loading 👍